skyportal/tests/frontend/test_newsfeed.py

```diff
@@ -85,10 +85,8 @@ def test_news_feed_prefs_widget(
     driver.get(f'/become_user/{user.id}')
     driver.get('/')
     # Default is to not show bot comments; enable for now
-    driver.click_xpath('//*[@id="newsFeedSettingsIcon"]', scroll_parent=True)
```

Comment:
I think they were not necessary, and at least on my machine, often got in the way. The screen would scroll down all the way instead of just finding the thing at the top of the screen. 
I think this tools should only be used in drop down menus (or maybe when you know the thing you are looking for is low on the page).

Reply:
How can you ever know where it's going to be on the page?  If it scrolls to the wrong position, please file a bug so we remember to track it.

Reply:
File a bug about the driver scrolling down too much?